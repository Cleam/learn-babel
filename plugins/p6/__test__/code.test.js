const babel = require('@babel/core');
const plugin = require('../babel-plugin-p6');

const srcCode = `function square(n) {
  return n * n;
}`;

it('works', () => {
  const { code } = babel.transformSync(srcCode, { plugins: [plugin] });
  expect(code).toMatchSnapshot();
});
