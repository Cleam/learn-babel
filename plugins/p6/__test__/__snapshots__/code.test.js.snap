// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`works 1`] = `
"function square(_n) {
  return _n * _n;
}"
`;
